pages/createOrder.js

```diff
@@ -117,6 +118,20 @@ class CreateOrderPage extends React.Component {
           </MessageBox>
         </Flex>
       );
+    } else if (!data.Tier) {
```

Comment:
This is good, but one path is not covered:

If you go to https://opencollective.com/apex/donate there's no tier but it's still valid (it's a custom donation). 

Please test it to ensure that it is correct, but I think you can cover everything by doing:

```suggestion
    } else if (this.props.tierId && !data.Tier) {
```

Reply:
I added it to the if condition and a screenshot to PR description.